presto-session-property-managers/src/test/java/io/prestosql/plugin/session/db/H2SessionPropertiesDao.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.session.db;
+
+import org.jdbi.v3.sqlobject.customizer.Bind;
+import org.jdbi.v3.sqlobject.statement.SqlUpdate;
+
+import static io.prestosql.plugin.session.db.util.SessionPropertiesDaoUtil.CLIENT_TAGS_TABLE;
+import static io.prestosql.plugin.session.db.util.SessionPropertiesDaoUtil.PROPERTIES_TABLE;
+import static io.prestosql.plugin.session.db.util.SessionPropertiesDaoUtil.SESSION_SPECS_TABLE;
+
+/**
+ * Provides additional methods to populate session specs and properties in the h2 database for testing purpose.
+ */
+public interface H2SessionPropertiesDao
+        extends SessionPropertiesDao
+{
+    @SqlUpdate("INSERT INTO " + SESSION_SPECS_TABLE + "(spec_id, user_regex, source_regex, query_type, group_regex, priority)\n" +
+            "VALUES (\n" +
+            "   :spec_id,\n" +
+            "   :user_regex,\n" +
+            "   :source_regex,\n" +
+            "   :query_type,\n" +
+            "   :group_regex,\n" +
+            "   :priority\n" +
+            ")")
+    void insertSpecRow(@Bind("spec_id") long specId,
```

Comment:
Nit: put the first parameter on the next line, so they are all aligned. Also, our code style is to indent continuation by 8 spaces, rather than aligning to the opening parenthesis.